src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -277,6 +268,27 @@ const ReleaseContent = () => {
                 )}
               </dd>
             </dl>
+            {allMethodologies.length > 0 && (
+              <>
+                <h2
```

Comment:
Should this be a h3 instead as it looks like it's under the 'Useful info' heading?

Reply:
Hey @amyb-hiveit I followed the convention that was there already for "Related pages" but my initial gut *was* to make it an h3 until I saw the Related pages header

Reply:
Spoke with Marv and confirmed that indeed they should both be h3s as you suggested, so changed them both, ta!